cartridges/openshift-origin-cartridge-haproxy/bin/control

```diff
@@ -147,15 +147,19 @@ function _stop_haproxy_service() {
     else
         if `pgrep -x haproxy > /dev/null 2>&1`
         then
-            echo "Warning: HAProxy process exists without a pid file.  Use force-stop to kill." 1>&2
+            client_message "Warning: HAProxy process exists without a pid file."
```

Comment:
Oh, there isn't a `client_warn` (there is a `warning`, but it doesn't prepend "Warning:" or "CLIENT_" anything).